### DISCUSSION TITLE
Player Balance

### DISCUSSION THREAD

**Discussion by hotspuraz** [Zrips/CMI#4697] [Feature Requests]
**Description of feature/improvement request:** 
Would it be possible to hide inactive players from /baltop until they log in again? 

---
### Detailed explanation of what it should do, when and in what situation: ### 
When a player is inactive for a set amount of time, their balance becomes hidden on the leaderboards.

### Basic explanation as to why this feature is needed and/or useful to have: ### 
I have a heavily economy driven server which many players like to compete to become the richest. However, many players leave the server after a while but their balance persists forever making it increasingly harder for newer players to climb up the leaderboards.

**Comment by LogGits** [comment]
I like this idea alot. If this is added we need to have a setting to determine how long is considered inactive (whereby 0 would disable this feature). Great suggestion.

**Comment by mrfloris** [comment]
While I like the idea, I do not think i'd implement it. Someone who's inactive simply has that balance, it's a truth. If you hide it and someone who's active climbs the ladder and makes it to the top, and the inactive player re-joins the server, poof, they are back at the top and that player who was active will probably not feel too happy about that. There's no hidden data if it's already there. It's just part of the economy and simply something to aim for. 

Again, I like the idea, but I doubt I'll turn it on on my servers
